news_items/043_units_element/index.html

```diff
@@ -0,0 +1,68 @@
+<style>
+  img {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 15px;
+    margin-bottom: 15px;
+  }
+
+  pre {
+    width: 90%;
+    margin-left: auto;
+    margin-right: auto;
+    overflow: hidden;
+    margin-bottom: 1rem;
+    font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
+    background: #f0f0f0;
+    padding: 1em;
+    padding-bottom: 0em;
+  }
+
+</style>
+
+<p>
+  PrairieLearn now has a new <code class="user-output">pl-units-input</code> element, due to
+  <a href="https://github.com/eliotwrobson">Eliot Robson</a> (UIUC).
```

Comment:
It sounds a bit weird if you thank yourself when you are the author of the news item...

Reply:
tbh, agreed, though I think we've done this before in other news items. IMO news items shouldn't have associated authors, and we can credit folks inline (like this) if we/they want.

Reply:
https://github.com/PrairieLearn/PrairieLearn/pull/7302#discussion_r1127250220

Reply:
tbh I kinda just copied what the other news items did when I wrote this.